superset/commands/base.py

```diff
@@ -70,3 +70,17 @@ def populate_owners(owner_ids: Optional[list[int]] = None) -> list[User]:
         :returns: Final list of owners
         """
         return populate_owners(owner_ids, default_to_user=False)
+
+    @staticmethod
+    def update_owner_list(
+        current_owners: list[User],
+        new_owners: Optional[list[int]],
+    ) -> list[User]:
+        """
+        Handle list of owners for update events.
+
+        :param current_owners: list of current owners
+        :param new_owners: list of new owners specified in the update payload
+        :returns: Final list of owners
+        """
+        return update_owner_list(current_owners, new_owners)
```

Comment:
I had that same reaction when started looking into this haha I kept it this flow because this was how `populate_owners` was imported. I'll implement your first suggestion in a future commit

Reply:
ended up renaming the functions to avoid importing `utils` entirely.